client/src/app/app.component.html

```diff
@@ -1,25 +1,15 @@
 <div>
   <section>
     <nav class="navbar navbar-expand-lg navbar-dark border-light">
-      <a class="navbar-brand" href="#">{{ title }}</a>
+      <a class="navbar-brand" [routerLink]="['/']">{{ title }}</a>
       <div class="collapse navbar-collapse">
-        <ul ngbNav #nav="ngbNav" [(activeId)]="active" class="navbar-nav">
-          <li ngbNavItem="employees">
-            <a ngbNavLink>Employees</a>
-            <ng-template ngbNavContent>
-              <app-employee></app-employee>
-            </ng-template>
-          </li>
-          <li ngbNavItem="payrolls">
-            <a ngbNavLink>Payrolls</a>
-            <ng-template ngbNavContent>
-              <app-payroll></app-payroll>
-            </ng-template>
+        <ul ngbNav [activeId]="fragment | async" class="navbar-nav">
+          <li [ngbNavItem]="link.path" *ngFor="let link of links">
+            <a ngbNavLink [routerLink]="[link.path]">{{ link.title }}</a>
           </li>
         </ul>
       </div>
     </nav>
-    <div [ngbNavOutlet]="nav"></div>
     <router-outlet></router-outlet>
```

Comment:
The components will now be displayed by the `<router-outlet>` instead of the `[ngbNavOutlet]`

Reply:
Interesting, I'm not really familiar with angular routing constructs yet so i'll have to look up more about this.